### ISSUE TITLE
Add GEXF 1.3 specifications

### ISSUE THREAD

**Issue by mbastian** [gephi/gexf#1]
Goals are 
1. to update the RelaxNG specifications so XSD files can be generated. Sources for 1.2 version are in specs folder.
2. update the PDF primer document. Latex sources are in the primer document.

**Comment by gvegayon** [comment]
Hey there,

I was wondering where is are the specifications =). Also, will there any work be done on the C++ library LibGEXF.

Thanks\

**Comment by sheymann** [comment]
https://gephi.org/gexf/format/

**Comment by duncdrum** [comment]
I have copied the (old) specs into the wiki of this repo, and started with updating them. Feel free to comment | contribute.

**Comment by gvegayon** [comment]
Great! Thanks

**Comment by kortschak** [comment]
What is the status of this? Will an xsd be released or alternatively if that is too far away, how accurate are the rnc snippets in the wiki wrt v1.3?

**Comment by kortschak** [comment]
*ping*

**Comment by eduramiba** [comment]
Hi,
This will be addressed with a new GEXF XSD but I will need to learn the tools and process to do that.

**Comment by kortschak** [comment]
Thanks. When you do that, please make the XSD minimally diff against the 1.2 definitions to allow ease of implementation from existing code (I'm working in Go which does not have good XSD handling tools - because thankfully it avoids XML wherever possible).

**Comment by mbastian** [comment]
Hi, and apologies for the long delay. I'm picking this up as the current situation isn't good.

Essentially, back in 2015/16, we've created a 1.3 semi-specification, based on the discussion at https://github.com/gephi/gephi/issues/1152 but haven't released any proper XSD or even RNC/RNG files. Some bits and pieces are located in this wiki, as a primer but it's incomplete. We've then adapted Gephi import/export to support this 1.3 specification starting in version 0.9. It hasn't changed much since but works fine. Some key 0.9. features like multi-graph or dynamic slicing rely on the GEXF 1.3 specification

Because of that we're created some confusion on what exactly is the 1.3 specification and which one we recommend using. In addition, it's a challenge for other community tools to follow the standard without proper documentation and a XSD. Ideally, other tools that import/export GEXF should support 1.3 as well.

To fix this, I'created a 1.3 branch and added some draft specifications. They are for the most part based on what Gephi currently implements but can still be extended based on your feedback.
